src/features/SendAssets/components/fields/GasPriceSlider.tsx

```diff
@@ -94,21 +50,52 @@ export default class SimpleGas extends Component<Props> {
       : {};
   };
 
-  private formatTooltip = (gas: number) => {
-    const { gasEstimates } = this.props;
-
+  const formatTooltip = () => {
     const gasTooltips: GasTooltips = {
       [gasEstimates.fast]: translateRaw('TX_FEE_RECOMMENDED_FAST'),
       [gasEstimates.fastest]: translateRaw('TX_FEE_RECOMMENDED_FASTEST'),
       [gasEstimates.safeLow]: translateRaw('TX_FEE_RECOMMENDED_SAFELOW'),
       [gasEstimates.standard]: translateRaw('TX_FEE_RECOMMENDED_STANDARD')
     };
 
-    const recommended = gasTooltips[gas] || '';
+    const recommended = gasTooltips[parseFloat(gasPrice)] || '';
 
     return translateRaw('GAS_GWEI_COST', {
-      $gas: gas.toString(),
+      $gas: gasPrice.toString(),
       $recommended: recommended
     });
   };
-}
+
+  const bounds = {
+    max: gasEstimates ? gasEstimates.fastest : GAS_PRICE_DEFAULT.max,
+    min: gasEstimates ? gasEstimates.safeLow : GAS_PRICE_DEFAULT.min
+  };
```

Comment:
Shouldn't the defaults be the responsibility of the Api handler or the Form ?
In any case, `gasEstimates`, is a required prop so we can replace intermediate variable with argument defaults.